app/index.js

```diff
@@ -284,7 +293,7 @@ module.exports = generators.Base.extend({
       this.package.scripts.prelint = 'npm run pretty'
       this.package.scripts.pretty = 'prettier-standard index.js src/**/*.js test/**/*.js bin/**/*.js'
 
-      if (this.mocha) {
+      if (this.mocha || this.jest) {
         this.package.standard = this.fs.readJSON(this.templatePath('linter/standard.json'))
```

Comment:
hmm probably now we need to split `linter/standard.json` in two files:

- one file for mocha globals: `linters/mocha-standard-globals.json`
- one file for jest globals: `linters/jest-standard-globals.json`